lib/emscripten/src/lib.rs

```diff
@@ -1056,10 +1056,13 @@ pub fn generate_emscripten_env(globals: &mut EmscriptenGlobals) -> ImportObject
 pub fn nullfunc(ctx: &mut Ctx, _x: u32) {
     use crate::process::abort_with_message;
     debug!("emscripten::nullfunc_i {}", _x);
-    abort_with_message(ctx, "Invalid function pointer. Perhaps this is an invalid value \
```

Comment:
Pretty sure this change is just `cargo fmt`. Either remove it or move it to a different PR.